Loop/Extensions/UIAlertController.swift

```diff
@@ -72,35 +72,35 @@ extension UIAlertController {
     /// Initializes an action sheet-styled controller for selecting a CGMManager
     ///
     /// - Parameters:
-    ///   - cgmManagers: An array of CGMManager-conforming types
+    ///   - cgmManagers: An array of CGMManagers
     ///   - pumpManager: A PumpManager/CGMManager combo instance
     ///   - selectionHandler: A closure to execute when either a new CGMManager or the current PumpManager is selected
-    ///   - cgmManager: The selected CGMManager type
-    ///   - pumpManager: The selected PumpManager instance
-    internal convenience init(cgmManagers: [CGMManagerUI.Type], pumpManager: CGMManager?, selectionHandler: @escaping (_ cgmManager: CGMManagerUI.Type?, _ pumpManager: CGMManager?) -> Void) {
+    ///   - cgmManager: The selected CGMManager
+    ///   - pumpManager: The selected PumpManager
+    internal convenience init(cgmManagers: [AvailableDevice], pumpManager: CGMManager?, selectionHandler: @escaping (_ identifier: String, _ pumpManager: CGMManager?) -> Void) {
         self.init(
             title: NSLocalizedString("Add CGM", comment: "Action sheet title selecting CGM"),
             message: nil,
             preferredStyle: .actionSheet
         )
-
+        
         if let pumpManager = pumpManager {
             addAction(UIAlertAction(
                 title: pumpManager.localizedTitle,
                 style: .default,
                 handler: { (_) in
-                    selectionHandler(nil, pumpManager)
-                }
+                    selectionHandler("", pumpManager)
```

Comment:
Does the empty string mean 1) no selection, or 2) an empty string selection? (I've never been fond of using an empty string to indicate "no choice". It's along the lines of using zero to indicate "no choice", when in fact zero is a valid number.)

Reply:
I agree. Good catch. Will update to `String?` to allow return of `nil`